pkg/benchmarkjunit/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang:1.13-alpine AS builder
+
+WORKDIR $GOPATH/src/k8s.io/test-infra/
+
+COPY go.mod .
+COPY go.sum .
+
+RUN go mod download
+
+COPY pkg/benchmarkjunit/ pkg/benchmarkjunit/
+
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -mod=readonly -a -installsuffix cgo -o /app ./pkg/benchmarkjunit
+
+FROM golang:1.13-alpine
+
+COPY --from=builder /app /benchmarkjunit
```

Comment:
I'm not familiar with how Dockerfiles work with multiple `FROM`s, `COPY --from=`, or `FROM ... AS ...`, but this seems like overkill for wrapping a simple go binary in a container.
I think it would be simpler and clearer to build the go binary with a step in cloudbuild.yaml. Here is an example: https://github.com/kubernetes/test-infra/blob/7240a8ccbc6c8a56219b90e4292a9214797f1190/logexporter/cloudbuild.yaml#L2-L14

Reply:
Ah thanks for the example. Simplified!